README.md

```diff
@@ -43,24 +43,33 @@ n26:
   username: "john.doe@example.com"
   password: "$upersecret"
   login_data_store_path: "~/.config/n26/token_data"
+  mfa_type: "oob"
 ```
 
 or use environment variables:
 
 - `N26_USER`: username
 - `N26_PASSWORD`: password
 - `N26_LOGIN_DATA_STORE_PATH`: optional **file** path to store login data (recommended for cli usage)
+- `N26_MFA_TYPE`: `oob` will use the paired app as 2 factor authentication, `otp` will use SMS to the registered number.
```

Comment:
Imo these config values should not have the technical term in them. A dev or user should not have to know wha `oob` and `otp` means. To make it more obvious to users I would rather use values like: 

* `paired_phone`, `phone_app` or simply `app`
* `sms`

Thoughts on that @femueller ?

Reply:
I changed it to `app` and `sms` for now.